src/core/NearWalletSelector.tsx

```diff
@@ -71,7 +72,7 @@ export default class NearWalletSelector {
     return this.walletController.getAccount();
   }
 
-  on(event: EventList, callback: () => void) {
-    this.walletController.on(event, callback);
+  on(event: any, callback: () => {}) {
```

Comment:
We should use the `EventList` string literal type here so that users of the library will get autocompletion